### ISSUE TITLE
[Question] Inherit preferences

### ISSUE THREAD

**Issue by fbove** [EliotBerriot/django-dynamic-preferences#174]
Hello!

Is it possible/doable to implement preferences inheritance using this lib?

An example could be having Company Preferences and User per Company Preferences.
So if the user has an specified preference already set for a company, asking for them will retrieve that among others from the company it belongs. Does it make sense?

**Comment by EliotBerriot** [comment]
Yes, it's possible to do so, you'd only `@register` the child preference and not the parent :)
